### DISCUSSION TITLE
Archived jobs

### DISCUSSION THREAD

**Discussion by WadeRolfe** [stayonhire/feedback#24] [Feedback / Issues]
When we create a job in the system to use for forward planning of fleet, if that job does not proceed or is lost, we cannot delete the job only archive it and never get access again.

In the event that a few days or a week later the client wants to proceed for whatever reason, is there a way to get access to that job again without having to recreate another one? Is this something that can be done from an administrator or is it not possible?

**Comment by dylanhru** [comment]
Hi @WadeRolfe,

There currently is a method to change the Job Status to Unsuccessful, which may address your issue directly.

1. From the menu option of any job, there is an option aptly named 'Change job status':
<img width="573" alt="Screenshot 2023-08-04 at 8 07 40 pm" src="https://github.com/stayonhire/feedback/assets/34927957/5318b739-ec2b-49ef-8fe3-ed1d53283685">


2. Select 'Unsuccessful':
<img width="527" alt="Screenshot 2023-08-04 at 8 17 17 pm" src="https://github.com/stayonhire/feedback/assets/34927957/30b4f710-59df-45d1-a96e-a337a0a809d6">


3. The job will change status with a 'U' icon and be moved into 'All' jobs:
<img width="1511" alt="Screenshot 2023-08-04 at 8 08 48 pm" src="https://github.com/stayonhire/feedback/assets/34927957/c0bd289a-ee49-4967-acf7-9cd7feb0de84">


4. Your team can also then also 'Reopen' the job if it becomes required again:
<img width="365" alt="Screenshot 2023-08-04 at 8 19 11 pm" src="https://github.com/stayonhire/feedback/assets/34927957/17668c9c-6b1b-43a0-9d9b-3594591095e8">

Other notes:
I have recently had a verbal request also to add a 'Reason' to Unsuccessful jobs (at the change of status) for teams to better measure why they may be losing work later on.

If the above solution is not sufficient, I will leave this discussion issue open, so you can get notified when we make the Archive accessible, as both Tom and I agree with the notion that your team should be able to access it.

**Reply by WadeRolfe** [reply]
Thanks Dylan, that makes perfect sense. This will be perfect for what I am looking for.
